src/containers/home/index.tsx

```diff
@@ -27,6 +29,9 @@ const HomeContainer = styled.div`
 
 const Home: React.FC = () => {
   const { windowType } = useWindowDimensions();
+
+  const userName = 'Jack!';
```

Comment:
lol

can you leave a TODO: here? 
```suggestion
// TODO: connect to backend's userData route
  const userName = 'Jack!';
```